websites/A/Aniflix/dist/metadata.json

```diff
@@ -5,21 +5,23 @@
   },
   "service": "Aniflix",
   "description": {
-    "en": "Aniflix offers a variety of anime series. Also, every day many new anime episodes appear on GER Sub right after Japan release.",
-    "nl": "Aniflix biedt verschillende anime-series. Ook verschijnen er elke dag veel nieuwe anime-afleveringen op GER Sub direct na de Japanse release.",
-    "de": "Aniflix bietet dir eine Vielfalt an Anime Serien. Außerdem erscheinen bei uns jeden Tag viele neue Anime Folgen auf Ger Sub direkt nach Japan-Release.",
+    "en": "Aniflix offers over 1300 German Dubbed and Subbed anime free to watch. Furthermore each Day the Newest Anime Episodes are Released after their Japanese Release.",
+    "nl": "Aniflix biedt meer dan 1300 Duitse Dubbed en Subbed anime om gratis te bekijken.  Bovendien worden elke dag de nieuwste anime-afleveringen uitgebracht na hun Japanse release",
```

Comment:
in the NL description there is 2 spaces after the dot

Reply:
corrected it, thats what i get for blindly trusting a friend